### PR TITLE
AzureStack.Backup.Admin: Fix test case BackupTests

### PR DESCRIPTION
**Pull request by daozha**
## Description
There was a bug in BackupTests causing the recorded session not passing PowerShell tests. Fix the bug and record again.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupTests.cs
Comment (deathly809) [inline src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupTests.cs]: The point of this test was to just test getting a single backup.

**Comment by dsgouda** [comment]
Please update the PR and squash commits. @deathly809 will merge when you approve

**Comment by deathly809** [comment]
@dsgouda I don't have permissions to merge.

**Comment by dsgouda** [comment]
@deathly809 I meant, I shall merge once you approve 😄

**Review by deathly809 — APPROVED** [review]

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupTests.cs
Comment (daozha) [reply src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupTests.cs]: I see. But as PS test goes through each backup in the same case, I think we can align them so that session records can be reused.

**Pull request merged**
